src/components/ens-registration/ConfirmContent/WaitCommitmentConfirmationContent.tsx

```diff
@@ -50,7 +53,7 @@ const WaitCommitmentConfirmationContent = ({
         </Row>
       </Rows>
       <Inset bottom={isSmallPhone ? '30px' : undefined}>
-        <ButtonPressAnimation onPress={action}>
+        <ButtonPressAnimation onPress={speedUpEnabled ? action : () => null}>
```

Comment:
there's gonna be a gap of 5 secs where we know the tx was confirmed but we'll wait a bit before changing the step